dex/testing/btc/base-harness.sh

```diff
@@ -39,7 +41,7 @@ else
   mkdir -p "${BETA_DIR}"
 fi
 
-cd ${NODES_ROOT} && tmux new-session -d -s $SESSION
+cd ${NODES_ROOT} && tmux new-session -d -s $SESSION $SHELL
```

Comment:
Having discussed this quite a bit in matrix chat with @amassarwi, I just want to note that this is the `shell-command` argument of tmux's `new-session` and `new-window` commands.  Without this, it seems to use the user's *login* shell instead of the currently running shell.
And while tmux docs claim it will use the shell in the `SHELL` environment variable for `shell-command`, @amassarwi found that this was not accurate.